From 9c8b3ffa108a3b85c6f2b82a24565371f16c7659 Mon Sep 17 00:00:00 2001
From: Andrew Savchenko <bircoph@gmail.com>
Date: Fri, 4 Jan 2019 07:02:07 +0300
Subject: [PATCH 08/14] flags

(cherry picked from commit c46c9e03ba1464eecae7c974461c982ee00f5000)
(cherry picked from commit 08b0d7d40710ac49cd8f4904b0985c9e5cdd88c5)
---
 src/config/Makefile.config.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/config/Makefile.config.in b/src/config/Makefile.config.in
index c5837bc66..ab877ce6a 100644
--- a/src/config/Makefile.config.in
+++ b/src/config/Makefile.config.in
@@ -222,11 +222,11 @@ COMPILE_ET_C=${COMPILE_ET} -emit c
 COMMON_INCL=$(MODULE_INCLUDE) -I${TOP_OBJDIR}/src/config -I${TOP_INCDIR} \
 	    -I${srcdir} -I. 
 
-COMMON_CFLAGS=$(CFLAGS) ${DBG} ${OPTMZ} $(XCFLAGS) \
+COMMON_CFLAGS=$(CFLAGS) $(XCFLAGS) \
 	      ${COMMON_INCL} ${INCLUDE_LIBINTL} \
 	      $(CPPFLAGS_roken) $(ARCHFLAGS)
 
-COMMON_LDFLAGS=$(LDFLAGS) $(LDFLAGS_hcrypto) $(LDFLAGS_roken) $(DBG) $(OPTMZ)
+COMMON_LDFLAGS=$(LDFLAGS) $(LDFLAGS_hcrypto) $(LDFLAGS_roken)
 
 # LWP Flags
 LWP_CFLAGS=$(MODULE_CFLAGS) $(COMMON_CFLAGS)
-- 
2.34.1

